qmetaobject/src/lib.rs

```diff
@@ -195,6 +195,7 @@ pub use qttypes::*;
 pub use tablemodel::*;
 
 pub mod connections;
+pub mod core;
```

Comment:
not sure core is a good name for a module because of the core crate.   Maybe core_application or just application would be better.

Reply:
So one Rust module per "class" then? And what about Qt modules, i.e. QtCore?  I recall we never reached consensus on this topic either. I believe it is worth experimenting with both styles here and there, and see which one works best in the end.

How about _qtcore_ for now, and maybe refactor into _qtcore::core_application_ later if needed? At the very least, that would go in-line with _qtdeclarative_ and _qtquickcontrols2_ modules.

Reply:
yeah, i don't know if it makes sense to have one rust module per Qt library.  
maybe per group of class that belong together.

Reply:
It is hard, and probably meaningless, to try to define a group of related classes. That would be some opinionated and not well-known mapping which would need to be documented somewhere.

On the other hand, file per class looks reasonable, because (1) that's what Qt already does, and (2) classes have so called "related non-members" and associated enums that would fit in such rust file/module just fine.

If there is ever going to be too many wrappers, at that point it might be feasible to split into "qmetaobject_qt{core,declarative,…}" crates to take advantage of parallel compilation in multi-threaded environments.